.gitignore

```diff
@@ -2,5 +2,6 @@
 work/
 data/
 results/
-tests/test_data/ngi-bisulfite_test_set/
 .DS_Store
+tests/test_data
```

Comment:
Do we really need that here?

Reply:
I find it helpful.. I usually run the test profile locally in a subfolder called `tests` or `testing`. And doesn't do any harm?

Reply:
then only `tests` is needed, no need for `tests/test_data` right?

Reply:
but true, doesn't do any arm